src/pybind/mgr/progress/module.py

```diff
@@ -671,9 +707,9 @@ def _handle_ls(self):
             if len(self._completed_events):
                 # TODO: limit number of completed events to show
                 out += "\n"
-                for ev in self._completed_events:
-                    out += "[{0}]: {1}\n".format("Complete" if not ev.failed else "Failed",
-                                                 ev.twoline_progress())
+                for ghost_ev in self._completed_events:
```

Comment:
why shall we encode the type into the variable name?

Reply:
https://mypy.readthedocs.io/en/stable/common_issues.html#redefinitions-with-incompatible-types 

`ev` was first declared in line 688.